tests/interface/functors/functors.dl

```diff
@@ -14,8 +14,12 @@ A(@goo("1234",2)) :- true.
 
 .output A
 
+.decl I(x:number) inline
+I(1) :- 1 < 5.
```

Comment:
This seems incomplete. If you're testing the combination of inlining and functors, you should be combining inlining and functors, which doesn't happen here. This rule is equivalent to a fact, and it doesn't use functors.

Reply:
The purpose was to inline `I` in a rule that *did* use a user-defined functor, which I actually missed. I've added this now.